runtime/services/vuid/watchdog.js

```diff
@@ -5,7 +5,7 @@ var property = require('@yoda/property')
 var logger = require('logger')('watchdog')
 
 var WATCHDOG_NODE = '/dev/watchdog'
-var WATCHDOG_DISABLE = property.get('watchdog.disable', 'persist')
+var WATCHDOG_DISABLE = property.get('state.watchdog.disable', 'persist')
```

Comment:
🤔Not-related changes? I'd suggesting not changing this.

Reply:
It's related, the `watchdog` properties should use the same prefix.

Reply:
watchdog.disable option is more a config property but not state.

Reply:
Updated :)